transformers/test_data/config_values.go

```diff
@@ -1,21 +1,35 @@
 package test_data
 
-import "github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+)
 
 // This file contains "shortcuts" to some configuration values useful for testing
 
-func CatAddress() string  { return constants.GetContractAddress("MCD_CAT") }
-func FlapAddress() string { return constants.GetContractAddress("MCD_FLAP") }
+func CatAddress() string  { return checksum(constants.GetContractAddress("MCD_CAT")) }
+func FlapAddress() string { return checksum(constants.GetContractAddress("MCD_FLAP")) }
 func FlipAddresses() []string {
-	return constants.GetContractAddresses([]string{
+	var addressesResult []string
+	flipAddresses := constants.GetContractAddresses([]string{
 		"MCD_FLIP_ETH_A", "MCD_FLIP_ETH_B", "MCD_FLIP_ETH_C",
 		"MCD_FLIP_REP_A", "MCD_FLIP_ZRX_A", "MCD_FLIP_OMG_A", "MCD_FLIP_BAT_A", "MCD_FLIP_DGD_A", "MCD_FLIP_GNT_A",
 	})
+
+	for _, address := range flipAddresses {
+		addressesResult = append(addressesResult, checksum(address))
+	}
+	return addressesResult
+}
+
+func EthFlipAddress() string    { return checksum(constants.GetContractAddress("MCD_FLIP_ETH_A")) }
+func FlopAddress() string       { return checksum(constants.GetContractAddress("MCD_FLOP")) }
+func JugAddress() string        { return checksum(constants.GetContractAddress("MCD_JUG")) }
+func SpotAddress() string       { return checksum(constants.GetContractAddress("MCD_SPOT")) }
+func VatAddress() string        { return checksum(constants.GetContractAddress("MCD_VAT")) }
+func VowAddress() string        { return checksum(constants.GetContractAddress("MCD_VOW")) }
+func CdpManagerAddress() string { return checksum(constants.GetContractAddress("CDP_MANAGER")) }
```

Comment:
Are all of these used? Wondering if we might be able to delete some of them

Reply:
We should let them stay until we figure out how to nuke them to hell :( It's so backwards to have to construct functions for things that are already in your config